addons/binding/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -59,10 +58,11 @@
 import org.openhab.binding.nanoleaf.internal.model.ControllerInfo;
 import org.openhab.binding.nanoleaf.internal.model.Ct;
 import org.openhab.binding.nanoleaf.internal.model.Effects;
+import org.openhab.binding.nanoleaf.internal.model.Hue;
 import org.openhab.binding.nanoleaf.internal.model.IntegerState;
 import org.openhab.binding.nanoleaf.internal.model.On;
-import org.openhab.binding.nanoleaf.internal.model.Palette;

```

Comment:
This class is not used any more - you could remove it

Reply:
It might be worth keeping that Palette class as it is still part of the data model of the API and can still be used in the future. Also, the Write class still uses that.

Reply:
> Also, the Write class still uses that.

I've successfully tested the Write class without the Palette, but we may need it for futher enhancements of the binding. So let's keep it.